README.md

```diff
@@ -39,28 +39,28 @@ assert value == "hello world!"
 
 ## Development Setup
 
-0. Install Rust and Python
-1. Install `maturin` in order to build Ypy
+0. [Install Rust](https://www.rust-lang.org/tools/install) and [Python](https://www.python.org/downloads/) 
+  - Some alternative bootstrapping options are [pyenv](https://github.com/pyenv/pyenv) for Python and using [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) to [conda install rust](https://anaconda.org/conda-forge/rust)
```

Comment:
```suggestion
  - Some alternative bootstrapping options are [pyenv](https://github.com/pyenv/pyenv) for Python and using [mamba](https://github.com/conda-forge/miniforge) to [mamba install rust](https://github.com/conda-forge/rust-feedstock)
```

Reply:
Actually we should mention that we can install python and rust in a conda environment.